client/my-sites/plans-grid/hooks/npm-ready/use-is-large-currency.ts

```diff
@@ -1,12 +1,55 @@
 import { formatCurrency } from '@automattic/format-currency';
-import type { GridPlan } from 'calypso/my-sites/plans-grid/hooks/npm-ready/data-store/use-grid-plans';
 
+const LARGE_ADD_ON_CURRENCY_CHAR_THRESHOLD = 7;
 const LARGE_CURRENCY_CHAR_THRESHOLD = 6;
 const LARGE_CURRENCY_COMBINED_CHAR_THRESHOLD = 9;
 
 interface Props {
-	gridPlans: GridPlan[];
-	returnMonthly?: boolean;
+	prices?: number[];
+	isAddOn?: boolean;
+	currencyCode: string;
+}
+
+function getDisplayPrices( prices: number[], currencyCode: string ) {
```

Comment:
Let's optimize the return value with `useMemo` here (probably means turning this into hook-form), as it's always a different object now irrespective of `prices`.

Reply:
Resolved in https://github.com/Automattic/wp-calypso/pull/81743/commits/98b6a99e85fec65096ef28526a4a657b7cb94c12